packages/connect-form/src/constants/ssl-tls-options.ts

```diff
@@ -0,0 +1,5 @@
+export enum TLS_OPTIONS {
+  DEFAULT = 'DEFAULT',
+  ON = 'ON',
+  OFF = 'OFF',
+}
```

Comment:
So … not trying to make a big deal out of details here, but how would you feel about using `export type TLS_OPTIONS = 'DEFAULT' | 'ON' | 'OFF'` here, and then using the string literals everywhere? It’s definitely something I have a personal preference for, because TS does gives us essentially the same guarantees it gives for enums, and imo it’s a bit nicer to type/there is no need to look up the enum definition to figure out what the exact value is :)

Reply:
Sure thing, updated. I'll try and avoid the enum pattern in future, definitely is a bit less verbose without it and means we don't have to import everywhere.